tensorrt_llm/llmapi/llm_args.py

```diff
@@ -1820,6 +1822,81 @@ def _load_config_from_ckpt(self, ckpt_dir: Path):
                 moe_tp_size=moe_tp_size,
                 moe_ep_size=moe_ep_size)
 
+    def get_executor_config(
```

Comment:
This method is for `TorchLlmArgs`, right? If so, I suggest we move this `get_executor_config` to `TorchLlmArgs` class directly.

BTW, in the following PR, we will fully deprecate the `ExecutorConfig` related code path in `PyExecutor`, and this `get_executor_config` will be removed as well. Is my understanding correct?